src/vstsbuildstatus.ts

```diff
@@ -63,26 +62,37 @@ export class VstsBuildStatus {
             return;
         }
 
-        if (!this.activeDefinition) {
-            this.statusBar.displayInformation("Select build definition", "");
-
+        if (!this.activeDefinitions) {
```

Comment:
You need to extend this condition to handle the initial case where activeDefinitions is an empty array. It is no longer null initially. As a result line 72 fails because it expects an item at position 0.